wiki/Rules/Code_of_conduct_for_modding_and_mapping/en.md

```diff
@@ -4,58 +4,32 @@ The **code of conduct for modding and mapping** is a set of rules and guidelines
-
-- Treat all suggestions as valid and equally important until you have thoroughly inspected and tested them — even minor stylistic changes or alternative viewpoints can help improve the quality of a beatmap
-- As with making a mod post, be sure to explain yourself as much as possible. State what you changed, and if you didn't accept the suggestion for some reason, explain why and what you did instead.
-- When declining a suggestion, remain calm and professional, say what needs to be said in order to explain yourself and **leave it at that**. No matter how impolite, confusing or unhelpful the suggestion was, getting angry and responding with hostility, sarcasm, etc. will not help the situation.
-- [Beatmap Nominators](/wiki/People/Beatmap_Nominators) (BN) and [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team) (NAT) members will review previous mods before placing any icons and may want to initiate or revisit discussions during their checks. Any assistance you can provide as to why you did or didn't do something takes very little effort, and could be a huge help to all parties later on.
+It is a mapper's responsibility to keep up with the latest discussions in their beatmaps. Ignoring suggestions or not responding to them in a timely manner can discourage other modders from participating in the discussion.
 
-## Further exchange and discussion
+When responding to someone's suggestions, it is important to consider the following:
 
-Sometimes, a single mod and response isn't enough to bring all parties to agreement, or fully resolve an issue. Perhaps the mapper doesn't agree with a suggestion, and the modder isn't satisfied with their response. Alternatively, both parties might agree that although one option is better, it isn't the best option and could still be improved further with additional input. Continued feedback is the best way to improve a beatmap, so get people involved and talking to get as many different ideas, or see things from as many points of view as necessary.
+- **Treat all suggestions as valid and equally important.** Don't draw hasty conclusions about the validity of a suggestion without properly assessing it.
+- **Explain yourself as much as possible.** State what you changed, and if you didn't accept the suggestion for some reason, explain why and what you did instead. This will not only help the modder to learn, but also allow future mods to be tailored to your concept or style.
```

Comment:
```suggestion
- **Explain yourself clearly.** State what you changed, and if you didn't accept the suggestion for some reason, explain why and what you did instead. This will not only help the modder to learn, but also allow future mods to be tailored to your concept or style.
```
"as much as possible" feels like it's encouraging long posts when that's not really the point (or at least that's one way to easily misread it)